indico/modules/designer/client/js/index.js

```diff
@@ -33,6 +33,13 @@
 
   var DEFAULT_PIXEL_CM = 50;
 
+  // Should add "is_image" property in the json and the PlaceHolder
+  var imageTypes = ["ticket_qr_code","event_logo"];
+
+  function isImage(type) {
+    return imageTypes.indexOf(type) > -1;
```

Comment:
const isImage = type => imageTypes.includes(type);

(if you prefer keeping the `function` statement, still use `includes` instead of `indexOf`)

Reply:
I agree.
I was also thinking... in my plugin I have to add a 3rd Image object, that is really "UN custom": Band. What can I do to add it in the plugin without customising the entire index.js ? can I keep the imageTypes in a separate JS file? Or maybe add a suffix: like event_logo_image , ticket_qr_code_image ? Not elegant but...

Reply:
with https://github.com/indico/indico/pull/4445/files#r420941411 this won't be a problem